docs/software/other/mqtt.md

```diff
@@ -11,16 +11,17 @@ Meshtastic devices with wifi hardware (ESP32) are able to connect to an MQTT bro
 - Connecting your mesh to the official Meshtastic MQTT broker. This makes your devices appear on the world map, and provides a limited copy of your mesh traffic, translated into JSON.
 - Using a custom MQTT broker to bridge several mesh networks together, via the internet (or just a local IP network)
 - Using a custom MQTT broker and a translator program to decode the raw protobuf packets and translate them into a plain text form for use in other systems. eg plotting temperature readings in Grafana, or device positions in Traccar.
+- Using or emitting packets directly in/from smart home control software such as Home Assistant or other consumers that can work with JSON messages.
 
-When MQTT enabled, the Meshtastic device simply uplinks and/or downlinks every raw protobuf packet that it sees to the MQTT broker. All packets are sent to the broker, whether they originate from another device on the mesh, or the gateway node itself.
+When MQTT enabled, the Meshtastic device simply uplinks and/or downlinks every raw protobuf packet that it sees to the MQTT broker. In addition, some packet types are decoded or encoded from/to JSON messages for easier use in consumers. All packets are sent to the broker, whether they originate from another device on the mesh, or the gateway node itself.
 
 Packets may be encrypted. If you use the default meshtastic MQTT server, packets are always encrypted. If you use a custom MQTT broker (ie set `mqtt_server`), the `mqtt_encryption_enabled` setting applies, which by default is false.
 
 IMPORTANT: When MQTT is turned on, you are potentially broadcasting your entire mesh traffic onto the public internet. This includes messages and position information.
 
-### MQTT Topic
+### MQTT Topics
 
-The device will uplink and downlink packets to the `msh/` prefix:
+The device will uplink and downlink raw packets to the `msh/` prefix:
```

Comment:
maybe:

The device will uplink and downlink raw([protobuf](https://developers.google.com/protocol-buffers)) packets to the `msh/` prefix: